File: src/python/extras/config_file.py
import json,os
from __main__ import app


def read_properties(config_file="example.json"):
    with open(config_file, 'r') as f:
        #config = json.load(f)
        lines=f.readlines()
        return lines


@app.route("/config")
def read_config():
    config_file=os.getenv("CONFIG_FILE","example.json")
    resp=""
    for line in read_properties(config_file):
        resp+=f"""
        {line}
        <br>
        """
        print(line)
    return f"<div style='text-align: center;'> <h1 >All from json config file:<br> </h1> {resp}  <br>- Python </div>"

File: src/nodejs/extras/config_file.js
const readline = require('readline');

const fs = require('fs');

module.exports = function(app){

	app.get('/config', function(req, res){

		const configFile=process.env.CONFIG_FILE || "./example.json";
		const config = require(configFile);

		const data = fs.readFileSync(configFile, 'UTF-8');
		const lines = data.split(/\r?\n/);
		txt=""
		lines.forEach((line) => {
			console.log(line);
			txt+=`${line}<br>`
		});

		res.send(`<div style="text-align: center;"> <h1 >All from json config file: </h1><br> ${txt} <br><h3>- NodeJS</h3> </div>` );
	});

	//other routes..
}


